src/WebJobs.Script/Description/CSharp/FunctionValueLoader.cs

```diff
@@ -0,0 +1,51 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Azure.WebJobs.Script.Description
+{
+    internal sealed class FunctionValueLoader : Lazy<Task<MethodInfo>>, IDisposable
```

Comment:
Please put a descriptive comment on this class. From reading the code, I'm not sure exactly why it is needed.